documentation/setup.LinuxDesktop.md

```diff
@@ -0,0 +1,44 @@
+## Linux Desktop setup for Docker DSpace
+
+
+### Install Git
+
+_Instructions will vary based on your Linux distribution._
+_These instructions have not been tested.  Please report your results with these instructions._
+
+#### CentOS
+
+```shell
+sudo yum install git
+```
+
+#### Debian, Ubuntu
+
+```shell
+sudo apt-get install git
+```
+
+#### Debian
```

Comment:
```suggestion
#### Fedora
```
From my experience in Linux, Fedora is the distribution that uses `dnf`, whereas Debian uses `apt` as shown above